charts/rasa-x/templates/app-deployment.yaml

```diff
@@ -19,6 +19,9 @@ spec:
       labels:
         {{- include "rasa-x.selectorLabels" . | nindent 8 }}
         app.kubernetes.io/component: app
+        {{- if .Values.app.podLabels }}
+{{ toYaml .Values.app.podLabels | indent 8 }}
```

Comment:
I understand that the `nindent` function is the same as the `indent` function, but prepends a new line to the beginning of the string -- just wanna make sure we don't want a new line between the pod labels here 😊

Reply:
Exactly, "The nindent function is the same as the indent function, but prepends a new line to the beginning of the string."